docs/content/docs/dev/table/sql/create.md

```diff
@@ -890,3 +891,46 @@ Language tag to instruct Flink runtime how to execute the function. Currently on
 Specifies the list of jar resources that contain the implementation of the function along with its dependencies. The jar should be located in a local or remote [file system]({{< ref "docs/deployment/filesystems/overview" >}}) such as hdfs/s3/oss which Flink current supports. 
 
 <span class="label label-danger">Attention</span> Currently only JAVA, SCALA language support USING clause.
+
+{{< top >}}
+
+## CREATE MODEL
+```sql
+CREATE [TEMPORARY] MODEL [IF NOT EXISTS] [catalog_name.][db_name.]model_name
+  [(
+    { <input_column_definition> }[ , ...n]
+    { <output_column_definition> }[ , ...n]
+  )]
+  [COMMENT model_comment]
+  WITH (key1=val1, key2=val2, ...)
+
+<input_column_definition>:
+  column_name column_type [COMMENT column_comment]
+
+<output_column_definition>:
+  column_name column_type [COMMENT column_comment]
+```
+
+Create a model with optional input and output column definitions. If a model with the same name already exists in the catalog, an exception is thrown.
+
+**TEMPORARY**
+
+Create temporary model that has catalog and database namespaces and overrides models.
+
+**IF NOT EXISTS**
+
+If the model already exists, nothing happens.
+
+**Input/Output Columns**
+
+The input columns define the features that will be used for model inference. The output columns define the predictions that the model will produce. Each column must have a name and data type. The input and output columns
+are optional because it can be inferred from training data later if Flink supports model training. For existing models, they are required to be defined.
```

Comment:
When you say "later if Flink supports model training", do you mean that model training could be performed by the user at some later time, or that Flink might introduce this capability in some later offering. It is not clear.
without this context I do not understand what  "For existing models, they are required to be defined." means.